metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsExperiment.groovy

```diff
@@ -5,12 +5,20 @@ class MetaStatsExperiment {
     String type
     List<String> samples
     Map<String,String> properties
-    String preparationSample
 
-    MetaStatsExperiment(String type, List<String> samples, Map<String,String> properties){
+    MetaStatsExperiment(String type, Map<String,String> properties){
+        this.type = type
+        this.properties = properties
+    }
+
+    MetaStatsExperiment(String type, Map<String,String> properties, List samples){
         this.type = type
-        this.samples = samples
         this.properties = properties
+        this.samples = samples
+    }
+
+    def addSamples(List<String> sampleCodes){
+        this.samples = sampleCodes
```

Comment:
```suggestion
    def addSamples(List<String> sampleCodes){
        this.samples.append(sampleCodes)
```
```suggestion
    def setSamples(List<String> sampleCodes){
        this.samples = sampleCodes
```

Reply:
removed this method since it is implicitly contained